test/model/src/driver/debug_log_parser.cpp

```diff
@@ -447,6 +484,35 @@ debug_log_parser::apply(kv_transaction_ptr txn, const txn_timestamp &op)
     txn->set_commit_timestamp(op.commit_ts);
 }
 
+/*
+ * debug_log_parser::row_truncate --
+ *     Apply the given operation to the model.
+ */
+void
+debug_log_parser::apply(kv_transaction_ptr txn, const row_truncate &op)
+{
+    /* Handle metadata operations. */
+    if (op.fileid == 0) {
+        throw model_exception("Unsupported metadata operation: row_truncate");
+        return;
+    }
+
+    /* Find the table. */
+    kv_table_ptr table = table_by_fileid(op.fileid);
+
+    /* Parse the keys. */
+    data_value start;
+    data_value stop;
+
+    if (op.mode == WT_TXN_TRUNC_BOTH || op.mode == WT_TXN_TRUNC_START)
+        start = data_value::unpack(op.start, table->key_format());
+    if (op.mode == WT_TXN_TRUNC_BOTH || op.mode == WT_TXN_TRUNC_STOP)
+        stop = data_value::unpack(op.stop, table->key_format());
+
+    /* Perform the operation. */
+    table->truncate(txn, start, stop);
```

Comment:
Are we ignoring the returned value here on purpose? Coverity will raise a warning.

Reply:
That's not on purpose, thanks for catching that. I change it to throw an exception on error, as errors are at this point unexpected (if it's in the debug log, it better be correct...). I also fixed it for the other operations.